consumer/utils/errors.py

```diff
@@ -1,26 +1,29 @@
-import json
-import typing
+from fastapi import Request
+from starlette.exceptions import HTTPException
 
 from .general import CustomResponse
-from fastapi import Request
 
-from starlette.exceptions import HTTPException
 
 class CustomException(Exception):
     """
     Usage:
         raise CustomException(errors=["INVALID_TOKEN"], status_code=401)
     """
+
+    __slots__ = ["errors", "status_code"]
+
```

Comment:
How come this magic attribute was added? [For reference](https://book.pythontips.com/en/latest/__slots__magic.html) re:what it does as I have not used it in a while.

Reply:
Flake8 / Mypy was alerting that we were using a Class which only sets its attributes in the ____init__.py__. It is suggested to, if it is immutable (and not used as a `dict` by adding attributes later) to either use a `namedTuple()` or `__slots__`. Adding this little line **fixes** the attributes, reducing memory and making it an immutable object.